### PR TITLE
LPS-193368 | Master

### PR DESCRIPTION
**Pull request by kristinonias**
https://liferay.atlassian.net/browse/LPS-193368

cc: @patriciajeanperez

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsCompletedItems.testcase [hunk 1]
```diff
@@ -339,11 +339,30 @@ definition {
 	}
 
 	@description = "LPS-94146 - Verify that when there are no results for a filter selection, the table should display the respective empty state"
-	@ignore = "Test Stub"
 	@priority = 3
 	test TableDisplaysEmptyStateWhenThereIsNoResult {
 
-		// TODO LRQA-67647 TableDisplaysEmptyStateWhenThereIsNoResult pending implementation
+		task("Given a workflow"){
```
Comment (kristinonias) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsCompletedItems.testcase:345]: Added test from https://liferay.atlassian.net/browse/LRQA-67647 and added task blocks.

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward:force

**Pull request closed without merging**